front/src/UI/organisms/AlarmChat/index.tsx

```diff
@@ -1,21 +1,41 @@
-import React, { useRef, MouseEvent } from 'react';
-
+import React, { useRef, MouseEvent, useEffect } from 'react';
 import { AccordionItem, AccordionButton, AccordionPanel, AccordionIcon, Box, Text, Flex } from '@chakra-ui/react';
+import { useQuery, useReactiveVar } from '@apollo/client';
 
 import { Menu } from '../ContextMenu';
-import { AlarmChatMessage } from '../../molecules';
+import { AlarmChatMessagesBox } from '../AlarmChatMessagesBox';
+import { ChatLogSendBox } from '../ChatLogSendBox';
+
 import { PersonIcon, LockIcon } from '../../../utils/icons';
-import { dummyChatData } from '../../../utils/dummy';
+import { compareTimeLapseToString, postgresTimeToDate } from '../../../utils/util';
+
 import {
   ALARM_TITLE_FONTWEIGHT,
   ALARM_TITLE_FONTSIZE,
   ALARM_CONTENT_FONTWEIGHT,
   ALARM_CHAT_TITLE_CONTENT_FONTSIZE,
   ALARM_BACKGROUND_COLOR,
 } from '../../../utils/constants';
+import { GET_CHAT, CHATLOG_SUBSCRIPTION } from './AlarmChatQueries';
+import { currentChatVar } from '../../../apollo/apolloProvider';
 
 export const AlarmChat = () => {
-  const { chat, chatLog } = dummyChatData;
+  const currentChat = useReactiveVar(currentChatVar);
+  const { loading, error, data, subscribeToMore } = useQuery(GET_CHAT, {
+    variables: {
+      uuid: currentChat,
+    },
+    fetchPolicy: 'network-only',
+  });
+
+  // subscription으로 데이터가 들어오면 스크롤을 아래로 이동
+  const scrollRef = useRef();
+  useEffect(() => {
+    if (scrollRef.current !== undefined && scrollRef.current !== null) {
```

Comment:
```suggestion
    if (!scrollRef.current) {
```

아마 해당 부분도 위에 타입과 초기값을 지정하지않아 발생한것으로 보입니다. 그것과는 별도로 위와 같이 표현해도 문제없을것같은데 혹시 어떠한가요?